### DISCUSSION TITLE
PiP

### DISCUSSION THREAD

**Discussion by imedfan** [minbrowser/min#2003] [Q&A] [answered]
Hello. Can you tell me how to make Picture-in-Picrure (for youtube )? I don't know js and can't make a script.

**Comment by PalmerAL** [comment, marked as answer]
This already exists, but it doesn't work very well because youtube hides the default context menu. You can access it by:

* Right-click on a youtube video
* Then right-click somewhere else on the video
* There should be a "Picture in Picture" item

**Reply by imedfan** [reply]
Thanks, it works!
